Library/OcCryptoLib/OcCryptoLib.inf

```diff
@@ -34,6 +34,7 @@
   Md5.c
   RsaDigitalSign.c
   Sha1.c
+  Sha2-avx.h
```

Comment:
I think EDK II prefers CamelCase names for the files as well, so better to have Sha2Avx.h.